tests/test_read_write/test_write_csv.py

```diff
@@ -0,0 +1,118 @@
+import pytest
+from clumper import Clumper
+
+
+def test_write_csv(tmp_path):
+    d = tmp_path / "sub"
+    d.mkdir()
+    path = d / "monopoly_copy.csv"
+    Clumper.read_csv("tests/data/monopoly.csv").write_csv(path)
+    reader = Clumper.read_csv(path)
+    assert Clumper.read_csv("tests/data/monopoly.csv").collect() == reader.collect()
+
+
+def test_write_csv_append_mode(tmp_path):
+    """Test that the append mode works."""
+    d = tmp_path / "sub"
+    d.mkdir()
+    path = d / "monopoly_copy.csv"
```

Comment:
This bit of code repeats itself in these tests. Might be good to write a helper function to create this pre-existing path.